db/migrate/20160830233558_migrate_ports_to_port_ranges_in_load_balancer_listener.rb

```diff
@@ -0,0 +1,26 @@
+class MigratePortsToPortRangesInLoadBalancerListener < ActiveRecord::Migration[5.0]
+  class LoadBalancerListener < ActiveRecord::Base
+  end
+
+  def up
+    LoadBalancerListener.where.not(:instance_port => nil).each do |x|
```

Comment:
this can be wrapped with 
say_with_time("Updating LoadBalancerListener ..") do
..
end